### ISSUE TITLE
Bug in Launching Sync Activity after Services Install

### ISSUE THREAD

**Issue by VarunT11** [opendatakit/tool-suite-X#234]
## Steps to Reproduce
- Uninstall the Existing Services Application and Install it again.
- Launch Services
- Click on Sync Option from Toolbar.

The Sync Activity Crashes on this Action.

## Stacktrace

```
java.lang.RuntimeException: Unable to resume activity {org.opendatakit.services/org.opendatakit.services.sync.actions.activities.SyncActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.opendatakit.sync.service.SyncAttachmentState.name()' on a null object reference
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4336)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4368)
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:52)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2123)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7710)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.opendatakit.sync.service.SyncAttachmentState.name()' on a null object reference
        at org.opendatakit.services.sync.actions.fragments.SyncFragment.getSyncAttachmentStateIndex(SyncFragment.java:480)
        at org.opendatakit.services.sync.actions.fragments.SyncFragment.onResume(SyncFragment.java:222)
        at androidx.fragment.app.Fragment.performResume(Fragment.java:3039)
        at androidx.fragment.app.FragmentStateManager.resume(FragmentStateManager.java:607)
        at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:306)
        at androidx.fragment.app.SpecialEffectsController$FragmentStateManagerOperation.complete(SpecialEffectsController.java:742)
        at androidx.fragment.app.SpecialEffectsController$Operation.cancel(SpecialEffectsController.java:594)
        at androidx.fragment.app.SpecialEffectsController.forceCompleteAllOperations(SpecialEffectsController.java:329)
        at androidx.fragment.app.SpecialEffectsController.executePendingOperations(SpecialEffectsController.java:264)
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2202)
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2100)
        at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:2002)
        at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3136)
        at androidx.fragment.app.FragmentManager.dispatchResume(FragmentManager.java:3084)
        at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:273)
        at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:458)
        at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:447)
        at androidx.appcompat.app.AppCompatActivity.onPostResume(AppCompatActivity.java:204)
        at android.app.Activity.performResume(Activity.java:7982)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4326)
```

## Solution for Fixing the Bug
- Checking if `syncAttachmentState` is not `null` at [SyncFragment.java:480](https://github.com/odk-x/services/blob/1a0b8db2b52a6caf9581dc8ab68a7e6e22ff413d/services_app/src/main/java/org/opendatakit/services/sync/actions/fragments/SyncFragment.java#L480) fixes this Issue!

**PS**: I would like to work on fixing this bug!

**Comment by wbrunette** [comment]
@VarunT11 I have assigned the issue to you.

**Comment by linl33** [comment]
Fixed by odk-x/services#201
